src/middlewares/enricher.ts

```diff
@@ -0,0 +1,52 @@
+import { Context, Next } from 'koa';
+import {
+  ProcessorTransformationRequest,
+  RouterTransformationRequest,
+  RouterTransformationRequestData,
+} from '../types';
+import GeoLocationHelper from '../helpers/geoLocation';
+
+export type DTRequest = RouterTransformationRequest | ProcessorTransformationRequest[];
+
+export default class Enricher {
+  private static enrichWithGeoInfo(
+    data: RouterTransformationRequestData[],
+  ): RouterTransformationRequestData[] {
+    return data.map((inpEv) => {
```

Comment:
acronyms like `inpEv` might not be readable, let's name them as `input` or `event` or `inputEvent`

Reply:
Incorporated